packages/models/src/store/store.ts

```diff
@@ -69,24 +70,51 @@ export class Store<
   }
 
   constructor(
-    schemaOption: GetStorageOption<StructSchema>,
+    schemaOption?: GetStorageOption<StructSchema>,
```

Comment:
> Why change `schemaOption` to be optional

When I use the first design, I changed this to optional because I want to pass the parameter `ref`. After I changed the design, the `ref` did not exist. But I noticed that the property `schemaOption` is optional and there is no other code to use this parameter, so I keep this change stay.

Reply:
Got it.

But it's weird to set `schema` optional and return `{}` if `schema` is `undefined`.

We can discuss it in another issue.

Reply:
The optional declaration will be removed by https://github.com/ckb-js/kuai/pull/240

Reply:
> The optional declaration will be removed by #240

restored to required